r/R/type.R

```diff
@@ -107,17 +111,48 @@ UInt32 <- R6Class("UInt32", inherit = FixedWidthType)
 UInt64 <- R6Class("UInt64", inherit = FixedWidthType)
 Float16 <- R6Class("Float16", inherit = FixedWidthType)
 Float32 <- R6Class("Float32", inherit = FixedWidthType)
-Float64 <- R6Class("Float64", inherit = FixedWidthType)
+Float64 <- R6Class("Float64",
+  inherit = FixedWidthType,
+  public = list(
+    code = function() call("float64")
+  )
+)
```

Comment:
Just wondering; because we support using `double()` to create `float64` objects, could this potentially lead to confusion for users, e.g. 

```
s <- schema(x = double())
s
# Schema
# x: double
s$code()
# schema(x = float64())
```

I think this perhaps doesn't matter though, given in the docs we do say "When called inside an arrow function, such as schema() or cast(), double() also is supported as a way of creating a float64()"

Reply:
ooh nice. I did not know that. using `double()` now.

Reply:
Yeah, that's why I was wondering if it was something to change or not - I think it's inevitably a little confusing either way! Perhaps having it show `float64()` as it was before would be better as it matches the corresponding C++ function?

Reply:
reverted to `float64()`